### ISSUE TITLE
Awesome new feature number 3

### ISSUE THREAD

**Issue by hvdklauw** [StriatumVentures/ZenHubTest#9]

**Comment by hanneman** [comment]
2 epics...?

**Comment by hanneman** [comment]
> Erik Huisman [2:12 PM]  als je geen storypoints toekent kan je ook geen burndown draaien
> Harro van der Klauw [2:12 PM] Hoeft ook niet van mij

We zouden toch estimates toe kunnen kennen als je de issue oppakt?

**Comment by hvdklauw** [comment]
Ja, maar eigenlijk moet je die dan ook aanpassen als je langer/korter bezig bent en komt er alleen maar heel veel management achter weg om de burndown te corrigeren

**Comment by hanneman** [comment]
ook waar.... + is het vooral interessant als je het zou willen doen voor het inplannen van de sprint... toch?
